studies/forms.py

```diff
@@ -322,6 +327,7 @@ class Meta:
                 For more information on experiment runner types, please
                 <a href={STUDY_TYPE_HELP_LINK}>see the documentation.</a></p>""",
             "structure": PROTOCOL_HELP_TEXT_INITIAL,
+            "priority": f"This affects how studies are ordered at your lab's custom URL, not the main study page. If you leave all studies at priority 99, all of your lab's active/discoverable studies will be shown in a randomized order on your lab page. If you set the priority of this study to 1, then it will appear last in the list on your lab page. You can find your lab's custom URL from the <a href='/exp/labs/'>labs page</a>. For more info, see the documentation on <a href='https://lookit.readthedocs.io/en/develop/researchers-manage-org.html#ordering-studies-on-your-lab-page'>study prioritization</a>.",
```

Comment:
Looks great, thanks for adding the links.
I think there's still a discrepancy between this part of the form and the way that it actually works. Specifically I think the 'Highest' and 'Lowest' labels on the slider need to be the other way around, so that the left is labelled 'Lowest' and the right is labelled 'Highest'.

And to make the mapping between values and priority a little clearer, I wonder if we can edit the text like this:

> If you leave all studies at the highest priority (99), then all...

and

> If you lower the priority of this study to 1, then...